app/index.html

```diff
@@ -4,14 +4,28 @@
 	<!-- bower:css -->
 	<!-- endbower -->
 	<link rel="stylesheet" href="styles/main.css"/>
+
+    <link rel="stylesheet" href="../bower_components/leaflet/leaflet.css" />
+    <script src="../bower_components/leaflet/leaflet.js"></script>
```

Comment:
We decided to use MapBoxGL JS instead of Leaflet, so these script/css imports are not necessary. Also, we do have the bower imports in the end of the body tag (which were removed here).